src/data/scripts/campaign/fleets/VayraPopularFrontManager.java

```diff
@@ -17,6 +17,7 @@
 import com.fs.starfarer.api.util.WeightedRandomPicker;
 import data.scripts.campaign.colonies.VayraColonialManager;
 import data.util.LoggerLogLevel;
+import data.util.Optional;
```

Comment:
consider introducing a similar "migration" mechanic here

Reply:
also, figure out what this `private final IntervalUtil timer = new IntervalUtil(30f, 60f);` does, and why it's hardcoded to 30-60.

Consider extracting that into LunaSettings, via a PopularFrontManagerExternalDataHolder or some such.

Reply:
done.

Reply:
resolving